package.json

```diff
@@ -72,21 +72,21 @@
   "devDependencies": {
     "@hint/configuration-all": "1.0.13",
     "cross-env": "6.0.3",
-    "eslint": "6.6.0",
+    "eslint": "6.7.2",
     "eslint-config-airbnb-base": "14.0.0",
-    "eslint-config-prettier": "6.5.0",
+    "eslint-config-prettier": "6.7.0",
     "eslint-plugin-import": "2.18.2",
     "eslint-plugin-prettier": "3.1.1",
     "hint": "5.4.1",
     "husky": "3.1.0",
     "jest": "24.9.0",
     "jest-fetch-mock": "2.1.2",
     "nock": "11.7.0",
-    "nodemon": "1.19.4",
+    "nodemon": "2.0.1",
```

Comment:
I think we can rip this out completely, we don't (or shouldn't be) using `nodemon` anymore.  Please also get rid of the line above that uses it, `"debug-server": "nodemon server.js"`